### DISCUSSION TITLE
Is FDS validated for fire suppression?

### DISCUSSION THREAD

**Discussion by CarloPollo** [firemodels/fds#12857] [Q&A]
Good morning,

Are there currently validated models for complete fire suppression involving commodities in logistics with stacking heights of 10m?

Can I use FDS for such an application without an experimental campaign?

Thaks to all

**Comment by drjfloyd** [comment]
No. The interaction of water droplets with burning commodity is an extremly complex problem.

**Reply by CarloPollo** [reply]
Thank you drjfloyd, as I imagined. I tried to look for such information on the FDS_Validation_Guide but didn't find anything. Is it possible or I didn't look properly?

Is there a chapter where the limitations of fds are listed?

**Reply by drjfloyd** [reply]
There aren't any examples of suppressing commodity in the Validation Guide. In general the Validation Guide is a good summary of types of problems that FDS has a reasonable capability to address. We don't at this time have detailed models for how water moves over and through objects. We can predict when sprinklers activate, where the water first impacts, but dealing with the movement of water over some complex 3D arrangment of objects (which in the case of commodity is often porous to some extent) is not something we have models for.

**Comment by mcgratta** [comment]
UL/NFPRF includes 5 rack storage suppression experiments

**Reply by drjfloyd** [reply]
Those relied on suppresion measurements made for similar arrangments of commodity with similar nozzles. I would not consider those to be generic inputs especially for the large distribution warehouses now being built with much larger stack heights.

**Reply by drjfloyd** [reply]
What is in the validation guide doesn't show any comparison on the prediction of heat releaese rate, fire extent, or success/failure of fire control. It is sequence of sprinkler operation and first activation time.

**Comment by CarloPollo** [comment]
Thanks to all for your replys

CP
